k8s_itests/deployments/paasta/fake_soa_config/compute-infra-test-service/deployments.json

```diff
@@ -1,20 +1,24 @@
 {
     "v1": {
-        "compute-infra-test-service:<%cluster%>.autoscaling": {
-            "docker_image": "services-compute-infra-test-service:paasta-a0ec9b362e29123022e5cebb6612aedc58a0889b",
+        "compute-infra-test-service:paasta-kind-emanelsabban-k8s-test.autoscaling": {
```

Comment:
should this be hardcoded?

Reply:
Thanks! Will change it. This file is also automatically generated when running ``generate_deployments_for_service.py``. So, if I changed  it to  <%cluster%>, the user can replace it manually with their cluster name or run "Generate deployments.json" in launch.json to generate this file  - ``deployments.json`` - automatically.